saleor/graphql/utils.py

```diff
@@ -8,13 +8,17 @@
 from graphql_jwt.utils import jwt_payload
 from graphql_relay import from_global_id
 
-from .core.enums import PermissionEnum, ReportingPeriod
+from .core.enums import OrderDirection, PermissionEnum, ReportingPeriod
 from .core.types import PermissionDisplay, SortInputObjectType
 
 ERROR_COULD_NO_RESOLVE_GLOBAL_ID = (
     "Could not resolve to a node with the global id list of '%s'."
 )
 registry = get_global_registry()
+REVERSED_DIRECTION = {
```

Comment:
This PR adds multiple utilities to this file which all are related to sorting. I think it may be a good time to reorganize it a bit and have:
```
utils/
  __init__.py
  sorting.py
```

And put all this sorting stuff to `sorting.py` while `__init__.py` would contain all the other functions taht are currently in `utils.py`. What do you think?

Reply:
sure, could point

Reply:
done